.github/workflows/build.yml

```diff
@@ -49,3 +46,6 @@ jobs:
 
       - name: Build parcel example
         run: yarn build:parcel
+
+      - name: Compare committed flow types to generated
```

Comment:
Moved this here so it's the last thing to run, as I wanted to get feedback for everything else first.